native/source/mixpanel/detail/mixpanel.cpp

```diff
@@ -32,15 +32,15 @@ namespace mixpanel
     using namespace detail;
 
     static std::chrono::steady_clock::time_point app_start = std::chrono::steady_clock::now();
-
-    std::shared_ptr<detail::Worker> worker;
+    
```

Comment:
Trailing whitespace; want to install the same commit hook we use in the backend repo? I can help with that.